modules/nw-ingress-sharding-route-labels.adoc

```diff
@@ -23,7 +23,7 @@ to another.
 
 . Edit the `router-internal.yaml` file:
 +
-[source,terminal]
+[source,yaml]
```

Comment:
this is misleading, the resulting _output_ is YAML, but the user _input_ is actually the `cat` command, which would use the terminal highlighting. not a blocking change, but should either be updated to remove the `cat` command to use the YAML highlighting, or should probably be changed back to terminal highlighting if that is not removed